src/gt4py/next/program_processors/runners/dace_common/workflow.py

```diff
@@ -25,22 +26,20 @@
 
 class CompiledDaceProgram(stages.CompiledProgram):
     sdfg_program: dace.CompiledSDFG
-    # Map SDFG argument to its position in program ABI; scalar arguments that are not used in the SDFG will not be present.
+    # Map SDFG positional arguments to the position in program ABI;
+    # scalar arguments that are not used in the SDFG will not be present.
     sdfg_arg_position: list[Optional[int]]
+    # Map arguments for connectivity tables to the position in program ABI;
+    # consider only the connectivity arrays, skip shape and stride symbols.
+    sdfg_conn_position: dict[str, int]
 
     def __init__(self, program: dace.CompiledSDFG):
-        # extract position of arguments in program ABI
-        sdfg_arglist = program.sdfg.signature_arglist(with_types=False)
-        sdfg_arg_pos_mapping = {param: pos for pos, param in enumerate(sdfg_arglist)}
-        sdfg_used_symbols = program.sdfg.used_symbols(all_symbols=False)
-
-        self.sdfg_program = program
-        self.sdfg_arg_position = [
-            sdfg_arg_pos_mapping[param]
-            if param in program.sdfg.arrays or param in sdfg_used_symbols
-            else None
-            for param in program.sdfg.arg_names
+        # method `signature_arglist` returns an ordered dictionary
```

Comment:
You should add what this mapping is? and it is `SDFG.arglist()`
Something like "`SDFG.arglist()` returns a mapping from the argument name to its type, the order is given by the ABI and the same as in `lastargs[0]`.